application/application.go

```diff
@@ -0,0 +1,101 @@
+package application
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/helper"
+	"github.com/josephroberts/edge-node-manager/proxyvisor"
+)
+
+type Application struct {
+	UUID       string
+	DeviceType *device.Type
+}
+
+func (a *Application) Process() error {
+	log.Printf("-----------------------------------------------------------------\r\n")
+	log.Printf("Application UUID: %s\r\n", a.UUID)
+	log.Printf("Application device type: %s\r\n", a.DeviceType.Device)
+	log.Printf("Application radio type: %s\r\n", a.DeviceType.Radio)
+
+	// Get latest firmware .zip and commit
+	application, commit, err := helper.GetApplication(config.GetPersistantDirectory(), a.UUID)
+	if err != nil {
+		log.Printf("No application location or commit found\r\n")
+	} else {
+		log.Printf("Application location: %s\r\n", application)
```

Comment:
Please don't add a carriage return :) this creates a horrible mess on non-windows systems. If you really want to be strict about line endings, use a helper function for this. I don't think this will be run on windows anyway?